src/nreplController.ts

```diff
@@ -46,13 +46,19 @@ const start = (): Promise<CljConnectionInformation> => {
         });
 
         nreplProcess.stderr.on('data', data => {
+            console.info('nrepl stderr =>', data.toString());
+        });
+
+        nreplProcess.on('exit', (code, signal) => {
+            console.info(`nREPL exit => ${code} / Signal: ${signal}`);
             stop();
-            return reject(`This error happened with our nREPL process: ${data}`);
+            return reject();
```

Comment:
We could reject the promise with an error string, then it'd show up in the vscode.window.showErrorMessage. This would make it easier both for users to understand what's going on, and for us to understand what's going wrong with bug reports in the future. Perhaps we could say here something like 'nREPL unexpectedly exited, error code ${code}'.